### DISCUSSION TITLE
Loading cursor when booting the computer for some minutes when placing it over the wallpaper, why?

### DISCUSSION THREAD

**Discussion by moshpirit** [i3/i3#6098] [Q&A]
If I move it to other apps, like Firefox, it stops, so I don't know why is appears to be loading anything. I use Nitrogen for wallpaper, if this helps.

[Screenshot](https://i.imgur.com/PH90YES.png)

[Config files that might be related](https://my.owndrive.com/s/LGsoARWmmPNJTEz) (including i3, of course).

**Comment by WaleedAmer** [comment]
Not sure if you've added any `exec`s in your i3 config, the link is no longer working.

I had this same problem from adding custom keybindings for launching Discord, Zen browser, etc, and it was caused from forgetting to add `--no-startup-id` after `exec`.

For example:
`bindsym $mod+Shift+d exec discord` <------------ Causes loading spinner
`bindsym $mod+Shift+d exec --no-startup-id discord`
